### PR TITLE
Dap sprint login setup

### PR DESCRIPTION
**Pull request by sainingo**
**In this branch I did the following:**
- Setup project requirements like - react-router, class to abstract localstorage methods.
- Login components

### CODE REVIEW

File: src/authentication/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                    <div>
+                        <button type="submit">Login</button>
+                    </div>
+                </form>
+            </div>
+        </div>
+    )
+}
+
+export default Login;
```
Comment (kipchubett) [inline src/authentication/Login.tsx]: Add new line at the end of every file

File: src/components/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Home = () => {
+  return (
+    <div>Home</div>
+  )
+}
+
+export default Home
```
Comment (kipchubett) [inline src/components/Home.tsx]: Ditto

File: src/app/localStorage.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    saveInfo = (data: {[key: string]: any}) => localStorage.setItem('userInformation', JSON.stringify(data))
+
+    loadData = () => localStorage.getItem('userInfo') || {}
+
+    getData = (key: string) => this.data[key]
+}
+
+
+const storage = new StoreData()
+export default storage;
```
Comment (kipchubett) [inline src/app/localStorage.tsx]: Why are using local storage?

**Review by kipchubett — COMMENTED** [review]
Thanks @sainingo 
Good initiative

File: src/app/localStorage.tsx (hunk 3 above)
Comment (sainingo) [reply src/app/localStorage.tsx]: To store abit of data about logged in user, eg role and later we can retrieve and use to perform other operations.

File: src/authentication/Login.tsx (hunk 1 above)
Comment (sainingo) [reply src/authentication/Login.tsx]: :+1:

**Pull request closed without merging**